server/views/pages/create/checkAnswers.test.ts

```diff
@@ -292,17 +292,6 @@ describe('Create a Licence Views - Check Answers', () => {
     expect($('#edit-licence-button-2').length).toBe(1)
   })
 
-  it('should show edit licence button when status is REJECTED', () => {
-    viewContext = {
-      licence: { ...licence, statusCode: 'REJECTED' },
-    }
-
-    const $ = cheerio.load(compiledTemplate.render(viewContext))
-
-    expect($('#edit-licence-button').length).toBe(1)
-    expect($('#edit-licence-button-2').length).toBe(1)
-  })
-
```

Comment:
Are we removing REJECTED as a thing? No way to REJECT currently, but I wondered if this would become a thing later? Fine for now.